src/middlewared/middlewared/plugins/disk_/mirror_base.py

```diff
@@ -0,0 +1,35 @@
+from middlewared.schema import accepts, Dict, Str, List
+from middlewared.service import filterable, private, ServicePartBase
+
+
+class DiskMirrorBase(ServicePartBase):
```

Comment:
I think it could be useful if we rename all mentions of disk mirror to disk swap mirror, otherwise feels like we are handling any regular mirrors for the system.